amqp/src/test/java/io/github/tcdl/msb/adapters/amqp/AmqpConsumerAdapterTest.java

```diff
@@ -227,6 +227,64 @@ public void testIsDurableTrueIfNotResponseTopicAndDurableConfig() throws IOExcep
         assertTrue(adapter.isDurable());
     }
 
+    @Test
+    public void testMessageCount() throws Exception {
+        String topicName = "myTopic";
+        String groupId = "groupId";
+        AmqpConsumerAdapter adapter = createAdapterWithNonDurableConf(topicName, groupId, false);
+
+        when(mockChannel.messageCount(anyString())).thenReturn(42L);
+
+        adapter.subscribe((jsonMessage, ackHandler) -> {
+        });
+
+        verify(mockChannel).exchangeDeclare(topicName, "fanout", false, true, null);
+
+        Optional<Long> answer = Optional.of(42L);
+        Optional<Long> result = adapter.messageCount();
+        verify(mockChannel, times(1)).messageCount(anyString());
+        assertEquals(result, answer);
```

Comment:
nit: Expected value should be the first argument in assertEquals.

Reply:
Indeed. Fixed now.

Reply:
@Mopper, Please fix argument order in other assert statements as well.